services/ppac/src/main/java/app/coronawarn/datadonation/services/ppac/config/SecurityConfig.java

```diff
@@ -40,17 +41,20 @@ public static LocalValidatorFactoryBean defaultValidator() {
   @Override
   protected void configure(final HttpSecurity http) throws Exception {
     http.authorizeRequests()
-        .mvcMatchers(HttpMethod.GET, HEALTH_ROUTE, PROMETHEUS_ROUTE, READINESS_ROUTE, LIVENESS_ROUTE).permitAll()
+        .mvcMatchers(HttpMethod.GET, HEALTH_ROUTE, PROMETHEUS_ROUTE, READINESS_ROUTE, LIVENESS_ROUTE)
+        .permitAll()
         .mvcMatchers(HttpMethod.POST, ANDROID + DATA, ANDROID + OTP, ANDROID + LOG).permitAll()
         .mvcMatchers(HttpMethod.POST, IOS + DATA, IOS + OTP, IOS + LOG).permitAll()
+        .mvcMatchers(HttpMethod.DELETE, DELETE_SALT).permitAll()
```

Comment:
can we make this profile depended?

Reply:
I don't know if it make sense, since the generate els token profile, for testing, is also part of this one